lib/rawscsi/request_signature.rb

```diff
@@ -1,9 +1,8 @@
 module Rawscsi
   class RequestSignature
-    def initialize(*args)
-      options = args.extract_options!
```

Comment:
@StevenJL  Yes, that's right, single `options` hash as params is ok. I think, `*args` is more of an artifact from the time, when I wasn't completely sure about the best way to pass data to `RequestSignature`.